client/actions/agenda.js

```diff
@@ -184,13 +184,49 @@ const _addToCurrentAgenda = (plannings) => (
 )
 
 /**
+ * Action factory that ask for creating a planning item from the supplied event,
+ * then calls addEventToCurrentAgenda
+ * @param {array} events - The events needed to create the planning items
+ */
+const askForAddEventToCurrentAgenda = (events) => (
+    (dispatch, getState) => {
+        const currentAgenda = selectors.getCurrentAgenda(getState())
+        if (currentAgenda) {
+            dispatch(showModal({
+                modalType: 'CONFIRMATION',
+                modalProps: {
+                    body: `Do you want to add these ${events.length}
```

Comment:
In case of a single event, it still mentions "events" and not "event". For a single event, it can probably just be "Do you want to add this event to the current agenda" ? That's probably more common.